src/pydata_sphinx_theme/assets/scripts/bootstrap.js

```diff
@@ -0,0 +1,16 @@
+/* Sphinx injects the html output with jquery and other javascript files.
```

Comment:
Can you add like a one sentence description of the file's purpose first? Something like `Import and setup functions to control Bootstrap's behavior`?